CompetitionTask/Pages/EducationFeature.cs

```diff
@@ -0,0 +1,265 @@
+        
+           IWebElement addUpdated;
+           private object language;
+           private double expectedLevel;
+
+
+        
+           public void EditEducation(String country, String universityName, String title, String degree, String year, String YearNew)
+           {
+
+              AddEducation(country,universityName, title, degree, year);
+
+            Thread.Sleep(12000);
+
+              editButton = _driver.FindElement(EditbtnLocator);
+              editButton.Click();
+
+               SelectElement yearDropdown = new SelectElement(_driver.FindElement(By.XPath("//select[@name='yearOfGraduation']")));
+               yearDropdown.SelectByText(YearNew);
+
+            // Optionally, save the changes by clicking the update button
+
+             addUpdated = _driver.FindElement(addUpdatedLocator);
+             addUpdated.Click();
+              Thread.Sleep(3000);
+        
+           }
+
+
+         public void VerifyEducationLeveleditted(String year, String expectedyear)
+           {
+            Thread.Sleep(7000);
+
+            // Create the XPath to find the eduction row
+            string xpath = $"//tbody/tr[td[text()='{expectedyear}']]/td[5]";
+         
+                       
+            IWebElement levelElement = _driver.FindElement(By.XPath(xpath));
+
+            Thread.Sleep(5000);
+
+               // Get the actual level text
+               string actualyear = levelElement.Text;
+
+            Thread.Sleep(5000);
+
+            // Verify the level
+            Assert.AreEqual(expectedyear, actualyear, $" Year of Graduation for was not updated correctly.");
+
+            Thread.Sleep(5000);
+
+            DeleteButton = _driver.FindElement(DeletebtnLocator);
+            DeleteButton.Click();
+
+        }
+
+                
+           public void DeleteEducation(String country, String universityName, String title, String degree, String year)
+           {
+                educationButton = _driver.FindElement(educationButtonLocator);
+                educationButton.Click();
+
+            AddEducation(country, universityName, title, degree, year);
+
+
+            Thread.Sleep(5000);
+
+              DeleteButton = _driver.FindElement(DeletebtnLocator);
+              DeleteButton.Click();
+
+            Thread.Sleep(5000);
+
+                          
+           }
+
+           public void VerifyEducationDeleted(string DeletedTitle)
+           {
+               Thread.Sleep(5000);
+               // Create the XPath to find the education row
+               string xpath = $"//tbody/tr[td[text()='{DeletedTitle}']]";
+               var educationElements = _driver.FindElements(By.XPath(xpath));
+               // Verify that no elements are found for the deleted education
+               Assert.IsTrue(educationElements.Count == 0, $"B.Sc '{DeletedTitle}' was not deleted successfully.");
```

Comment:
for future please move the asserts to steps

Reply:
Resolved